app-text/pandoc-cli/pandoc-cli-0.1.1.ebuild

```diff
@@ -14,7 +14,7 @@ HOMEPAGE="https://pandoc.org"
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE="+lua nightly +server"
+IUSE="+lua profile nightly +server"
```

Comment:
This needs to be added to CABAL_FEATURES rather than IUSE. We received a patch on this via email, I need to merge it in.